modules/nw-dual-stack-convert.adoc

```diff
@@ -0,0 +1,81 @@
+[id="nw-dual-stack-convert_{context}"]
+= Converting to a dual network stack cluster network
+
+As a cluster administrator, you can convert your single stack cluster network to a dual-stack cluster network.
+
+[NOTE]
+====
+After converting to dual-stack networking only newly created pods are assigned IPv6 addresses. Any pods created before the conversion must be recreated to receive an IPv6 address.
+====
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You are logged in to the cluster with a user with `cluster-admin` privileges.
+* Your cluster is using the OVN-Kubernetes cluster network provider.
+
+.Procedure
+
+. To specify IPv6 address blocks for the cluster and service networks, create a file containing the following YAML:
++
+--
+[source,yaml]
+----
+- op: add
+  path: /spec/clusterNetwork/-
+  value: <1>
+    cidr: fd01::/48
+    hostPrefix: 64
+- op: add
+  path: /spec/serviceNetwork/-
+  value: fd02::/112 <2>
+----
+<1> Specify an object with the `cidr` and `hostPrefix` fields. The host prefix must be `64` or greater. The IPv6 CIDR prefix must be large enough to accommodate the specified host prefix.
+
+<2> Specify an IPv6 CIDR with a prefix of `112`. Kubernetes uses only the lowest 16 bits. For a prefix of `112`, IP addresses are assigned from `112` to `128` bits.
+--
+
+. To patch the cluster network configuration, enter the following command:
++
+[source,terminal]
+----
+$ oc patch network.config.openshift.io cluster \
+  --type='json' --patch-file <file>.yaml
+----
++
+--
+where
+
+`file`:: Specifies the name of the file you created in the previous step.
+--
++
+.Example output
+[source,text]
+----
+network.config.openshift.io/cluster patched
+----
+
+.Verification
+
+. To verify that the cluster network recognizes the IPv6 address blocks that you specified in the previous procedure, enter the following command:
```

Comment:
My $0.02 is that it'll be better to add a one-liner paragraph to introduce the step.  Maybe something like "To verify that the conversion process succeeded, confirm that the network configuration includes the IPv6 addresses that you specified."

Then, the step 1 paragraph can be something like "Display the network configuration:"